docs/source/IC.rst

```diff
@@ -1,22 +1,100 @@
 IC
 =====
 
-IC stands for Invisible Cities
-*Language*: Python
+IC stands for *Invisible Cities* and can be found on Github as `next-exp/IC <https://github.com/next-exp/IC>`_.
 
-This section is focused on reviewing how the code is structured (:ref:`ICcode`)
-and its installation (:ref:`ICinstallation`).
+**Language**: Python; **Based**: Numpy, Pandas
+
+This section is focused on reviewing how the code is structured and where the installation guides can be found.
 
 .. _ICcode:
 
 Code
 ------------
 
-Info from code
+IC is a reconstruction software structured in algorithms/programs named *cities*. The code is written Python, but it uses `Numpy <https://numpy.org/>`_ and `Pandas <https://pandas.pydata.org/>`_ for data analysis and as a manipulation tool.
+Each of the *cities* is focused on different reconstruction tools of the code (e.g. raw waveforms integration, energy correction, deconvolution, etc...), and follow *dataflow* data processing: they resemble a pipeline of data transformations. More details about the dataformat can be found in a previous IC-crash-course in `Dataflow <https://github.com/mmkekic/IC-crash-course/blob/master/presentations/Dataflow.pdf>`_ section.
+A complete review about the IC repository can be found on a previous IC-crash-course under `IC-structure <https://github.com/mmkekic/IC-crash-course/blob/master/presentations/IC_structure.pdf>`_.
 
 .. _ICinstallation:
 
 Installation
 ------------
 
-Info about installation
+There is a *Quickstart guide* at the next-exp/IC repository with anything related to its installation. When compiling it, `conda <https://docs.conda.io/projects/conda/en/latest/index.html>`_ will be installed and it will be created the appropriate IC environment, as well as it will be set the environment variables which are needed for the correct functioning of IC.
+
+
+City Structure:
```

Comment:
This bullet has `:`, while the others don't. I would use the same convention for all of them (personally, I prefer no `:`).